### DISCUSSION TITLE
Discord server for Text Generator

### DISCUSSION THREAD

**Discussion by nhaouari** [nhaouari/obsidian-textgenerator-plugin#120] [Announcements]
Hello everyone,

I'm excited to announce the creation of our new Discord server! This is a space for us to connect, share our interests, and build a community.

We've created channels for general chat, introductions, announcements, events, feedback, templates, workflows, LLM, embeddings, and off-topic discussions. 

We hope you'll find something that interests you and join in on the conversation.

We're open to suggestions and feedback on how we can make this server even better, so please don't hesitate to let us know your thoughts.

To join, simply follow this link (new):
https://discord.gg/BRYqetyjag
We look forward to seeing you there!

Best regards,
Noureddine

**Comment by pdworkman** [comment]
Is there a new link? That one doesn't work for me

**Reply by nhaouari** [reply]
Thank you for reporting the issue. This is the new link for discord server : [https://discord.gg/BRYqetyjag](https://discord.gg/BRYqetyjag)

**Comment by dvansoye** [comment]
What do I need to do to chat in discord using this new server? I clicked in
the link, but I can send a message to anyone. Is there something I need to
accept first? Do I need to be approved?
